remove legacy statement on CSI upgrades (#894)
portworx/pxdocs: content/portworx-install-with-kubernetes/storage-operations/csi/_index.md

You may also use ephemeral inline volumes with deployments and other application types. See this [Kubernetes blog](https://kubernetes.io/blog/2020/01/21/csi-ephemeral-inline-volumes/) for more information and example use cases.

## Migration to CSI PVCs

> Currently, there is no support for migration/conversion from PVCs created using native Kubernetes driver to CSI driver. However, this is not required and both type of PVCs can co-exist in the same cluster.

Reviewer: ```suggestion
Currently, you cannot migrate or convert PVCs created using the native Kubernetes driver to the CSI driver. However, this is not required and both types of PVCs can co-exist on the same cluster.
```
